spring-ai-docs/src/main/antora/modules/ROOT/pages/api/chat/functions/azure-open-ai-chat-functions.adoc

```diff
@@ -1,5 +1,7 @@
 = Azure OpenAI Function Calling
```

Comment:
Consider adding `(Deprecated)` to the title like the other models already migrated